lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/dva/DvaThirdPartyDocumentGateway.java

```diff
@@ -167,6 +175,8 @@ public DocumentCheckResult performDocumentCheck(DrivingPermitForm drivingPermitD
 
         eventProbe.counterMetric(DVA_RESPONSE_TYPE_VALID.withEndpointPrefix());
 
+        documentCheckResult.setApiResultSource(API_RESULT_SOURCE);
```

Comment:
Can I just ask what is this doing? Is this just for discerning between dva/dcs/dvla?

Reply:
Due to the differences in what form values are used in the remote request for each api there is a discrepancy in the audit events/vc that may need to be fixed.

One approach was to record which form fields are used for each api remote request (in some sort of map - a bit convoluted) or to simply record the api used. 

If needed the enum can be used later after the verification result is returned to infer which fields where actually used in the each remote request (with out having to pass up a map of filenames) and should then be used generating the audit events/vc.